clients/cobol-lsp-vscode-extension/src/services/reporter/model/TelemetryEvent.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2020 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Broadcom, Inc. - initial API and implementation
+ */
+
+export class TelemetryEvent {
+    private readonly timestamp: string;
```

Comment:
I would suggest to [redesign](https://www.typescripttutorial.net/typescript-tutorial/typescript-getters-setters/) this private fields. In TS private fields usage is a bit different than Java. Another alternative is via constructor(public, .. ).  Thanks